willyserge/broadcaster
# #170049360  set up database environment

#### What does this PR do?
  Setting up the database environment
#### Description of Task to be completed?
- created models for creating tables and integrated them into the API
#### How should this be manually tested?
- run `npm install` to install all dependencies
- run `npm start `and to start the server
- check-in the console for the  `Initialized models` message
#### What are the relevant pivotal tracker stories?
[#170049360](https://www.pivotaltracker.com/n/projects/2418444/stories/170049360)
